### PR TITLE
Add indentation to the read me file

### PR DESCRIPTION
**Pull request by Smridhi-3**

### CODE REVIEW

**Comment by ErikBoesen** [comment]
Check your git config—your name needs to be your GitHub username for GH to recognize that you're the author of the commit. Note how the commits in this PR are associated with the name "Smridhi Madan" which is not recognized by GitHub. Instructions: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup

File: README.md [hunk 1]
```diff
@@ -7,20 +7,18 @@ This website uses the Jekyll templating system and is hosted as a GitHub page ou
...
-        gem install bundler jekyll --user
+gem install bundler jekyll
+or locally:
+ gem install bundler jekyll --user
 
 ## Setup
 1. While in cloned folder, run:
 
-        bundle install
+bundle install
```
Comment (ErikBoesen) [inline README.md]: No indentation should change here... looks like maybe some text editor tool you were using might have messed with this automatically?

File: _pages/team.html [hunk 2]
```diff
@@ -147,6 +147,12 @@ <h5>Ricardo Quiroz </h5>
             <h5>Brandon Werbel</h5>
             <p>Brandon is a new member of the team and is a programmer.</p>
         </div>
+        <div>
+            <img src="/images/team/placeholder.png">
+            <p class="label">🖥️ Programmer 🖥️</p>
+            <h5>Smridhi Madan</h5>
+            <p>Smridhi is a new member of the team and is a programmer.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Well done on this!

**Comment by AndrewLester** [comment]
@Smridhi-3 for some reason it seems to require a few more spaces as indentation. You can try previewing the file in VSCode to see what it would look like when formatted (ctrl + shift + v), or we can just try it another day.

**Pull request closed without merging**